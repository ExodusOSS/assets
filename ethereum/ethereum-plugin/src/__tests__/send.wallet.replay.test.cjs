jest.exodus.mock.fetchReplay()

import('./send.wallet.integration.test.js')
