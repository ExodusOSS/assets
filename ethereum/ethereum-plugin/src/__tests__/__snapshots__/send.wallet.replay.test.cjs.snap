exports[`ethereum send transfer integration test Send all bat token to second portfolio 1`] = `
{
  "1_amount": "4.42277112 BAT",
  "2_txId": "0xb0f511f2459b0bb03c899db2b24f6d90c1516a5f9597ad5019d406c2de04c795",
  "3_txRawHex": "02f8b00113841dcd6500850bc208d900828aa9940d8775f648430679a709e98d2b0cb6250d2887ef80b844a9059cbb000000000000000000000000f6c138c36341138ddfc314a11038da8264b7ef090000000000000000000000000000000000000000000000003d60d6e69a74e000c001a0263c888d759291318789bc9e8b73bda73eaecd9753db80beaf034d0882bdb453a07a41bdf6c0f6735d4502a2a0156dcc8a330889f198b8a7d812c881a15147dc17",
  "4_tx": {
    "coinAmount": "-4.42277112 BAT",
    "coinName": "bat",
    "confirmations": 0,
    "currencies": {
      "bat": {
        "BAT": 18,
        "base": 0,
      },
      "ethereum": {
        "ETH": 18,
        "Gwei": 9,
        "Kwei": 3,
        "Mwei": 6,
        "finney": 15,
        "szabo": 12,
        "wei": 0,
      },
    },
    "data": {
      "gasLimit": 35497,
      "nonce": 19,
      "replacedTxId": undefined,
      "tipGasPrice": "500000000",
    },
    "date": "SOME DATE",
    "feeAmount": "0.0017925985 ETH",
    "feeCoinName": "ethereum",
    "to": "0xf6c138c36341138ddfc314a11038da8264b7ef09",
    "txId": "0xb0f511f2459b0bb03c899db2b24f6d90c1516a5f9597ad5019d406c2de04c795",
    "version": 1,
  },
  "5_beforeBalances": {
    "exodus_0": {
      "bat": {
        "balance": "4.42277112 BAT",
        "spendable": "4.42277112 BAT",
        "spendableBalance": "4.42277112 BAT",
        "staked": "0 BAT",
        "staking": "0 BAT",
        "total": "4.42277112 BAT",
        "unconfirmedReceived": "0 BAT",
        "unconfirmedSent": "0 BAT",
        "unstaked": "0 BAT",
        "unstaking": "0 BAT",
      },
      "ethereum": {
        "balance": "0.004228230740891449 ETH",
        "spendable": "0.004228230740891449 ETH",
        "spendableBalance": "0.004228230740891449 ETH",
        "staked": "0 ETH",
        "staking": "0 ETH",
        "total": "0.004228230740891449 ETH",
        "unconfirmedReceived": "0 ETH",
        "unconfirmedSent": "0 ETH",
        "unstaked": "0 ETH",
        "unstaking": "0 ETH",
      },
    },
    "exodus_1": {
      "bat": {
        "balance": "0 BAT",
        "spendable": "0 BAT",
        "spendableBalance": "0 BAT",
        "staked": "0 BAT",
        "staking": "0 BAT",
        "total": "0 BAT",
        "unconfirmedReceived": "0 BAT",
        "unconfirmedSent": "0 BAT",
        "unstaked": "0 BAT",
        "unstaking": "0 BAT",
      },
      "ethereum": {
        "balance": "0.005932010192264297 ETH",
        "spendable": "0.005932010192264297 ETH",
        "spendableBalance": "0.005932010192264297 ETH",
        "staked": "0 ETH",
        "staking": "0 ETH",
        "total": "0.005932010192264297 ETH",
        "unconfirmedReceived": "0 ETH",
        "unconfirmedSent": "0 ETH",
        "unstaked": "0 ETH",
        "unstaking": "0 ETH",
      },
    },
  },
  "6_afterBalances": {
    "exodus_0": {
      "bat": {
        "balance": "0 BAT",
        "spendable": "0 BAT",
        "spendableBalance": "0 BAT",
        "staked": "0 BAT",
        "staking": "0 BAT",
        "total": "0 BAT",
        "unconfirmedReceived": "0 BAT",
        "unconfirmedSent": "4.42277112 BAT",
        "unstaked": "0 BAT",
        "unstaking": "0 BAT",
      },
      "ethereum": {
        "balance": "0.002435632240891449 ETH",
        "spendable": "0.002435632240891449 ETH",
        "spendableBalance": "0.002435632240891449 ETH",
        "staked": "0 ETH",
        "staking": "0 ETH",
        "total": "0.002435632240891449 ETH",
        "unconfirmedReceived": "0 ETH",
        "unconfirmedSent": "0.0017925985 ETH",
        "unstaked": "0 ETH",
        "unstaking": "0 ETH",
      },
    },
    "exodus_1": {
      "bat": {
        "balance": "0 BAT",
        "spendable": "0 BAT",
        "spendableBalance": "0 BAT",
        "staked": "0 BAT",
        "staking": "0 BAT",
        "total": "0 BAT",
        "unconfirmedReceived": "0 BAT",
        "unconfirmedSent": "0 BAT",
        "unstaked": "0 BAT",
        "unstaking": "0 BAT",
      },
      "ethereum": {
        "balance": "0.005932010192264297 ETH",
        "spendable": "0.005932010192264297 ETH",
        "spendableBalance": "0.005932010192264297 ETH",
        "staked": "0 ETH",
        "staking": "0 ETH",
        "total": "0.005932010192264297 ETH",
        "unconfirmedReceived": "0 ETH",
        "unconfirmedSent": "0 ETH",
        "unstaked": "0 ETH",
        "unstaking": "0 ETH",
      },
    },
  },
}
`;

exports[`ethereum send transfer integration test Send all eth from first to second portolio 1`] = `
{
  "1_amount": "0.004228230740891449 ETH",
  "2_txId": "0xea11a49d39a485942de652306c6a4c354d77d9217135bd39bd805dae97fed047",
  "3_txRawHex": "02f8720113841dcd6500850bc208d90082520894f6c138c36341138ddfc314a11038da8264b7ef09870f058daf56d73980c080a0863a7f3f37e85e68b51788faa2c31a72cf4e4d32469d48bec0f4aab0eeb6f19ea050af887d975b684b632ea378c58eb45f201d42bd1b33935def5a47ddd1f211d0",
  "4_tx": {
    "coinAmount": "-0.004228230740891449 ETH",
    "coinName": "ethereum",
    "confirmations": 0,
    "currencies": {
      "ethereum": {
        "ETH": 18,
        "Gwei": 9,
        "Kwei": 3,
        "Mwei": 6,
        "finney": 15,
        "szabo": 12,
        "wei": 0,
      },
    },
    "data": {
      "gasLimit": 21000,
      "nonce": 19,
      "replacedTxId": undefined,
      "tipGasPrice": "500000000",
    },
    "date": "SOME DATE",
    "feeAmount": "0.0010605 ETH",
    "feeCoinName": "ethereum",
    "to": "0xf6c138c36341138ddfc314a11038da8264b7ef09",
    "txId": "0xea11a49d39a485942de652306c6a4c354d77d9217135bd39bd805dae97fed047",
    "version": 1,
  },
  "5_beforeBalances": {
    "exodus_0": {
      "ethereum": {
        "balance": "0.004228230740891449 ETH",
        "spendable": "0.004228230740891449 ETH",
        "spendableBalance": "0.004228230740891449 ETH",
        "staked": "0 ETH",
        "staking": "0 ETH",
        "total": "0.004228230740891449 ETH",
        "unconfirmedReceived": "0 ETH",
        "unconfirmedSent": "0 ETH",
        "unstaked": "0 ETH",
        "unstaking": "0 ETH",
      },
    },
    "exodus_1": {
      "ethereum": {
        "balance": "0.005932010192264297 ETH",
        "spendable": "0.005932010192264297 ETH",
        "spendableBalance": "0.005932010192264297 ETH",
        "staked": "0 ETH",
        "staking": "0 ETH",
        "total": "0.005932010192264297 ETH",
        "unconfirmedReceived": "0 ETH",
        "unconfirmedSent": "0 ETH",
        "unstaked": "0 ETH",
        "unstaking": "0 ETH",
      },
    },
  },
  "6_afterBalances": {
    "exodus_0": {
      "ethereum": {
        "balance": "0 ETH",
        "spendable": "0 ETH",
        "spendableBalance": "0 ETH",
        "staked": "0 ETH",
        "staking": "0 ETH",
        "total": "0 ETH",
        "unconfirmedReceived": "0 ETH",
        "unconfirmedSent": "0.005288730740891449 ETH",
        "unstaked": "0 ETH",
        "unstaking": "0 ETH",
      },
    },
    "exodus_1": {
      "ethereum": {
        "balance": "0.005932010192264297 ETH",
        "spendable": "0.005932010192264297 ETH",
        "spendableBalance": "0.005932010192264297 ETH",
        "staked": "0 ETH",
        "staking": "0 ETH",
        "total": "0.005932010192264297 ETH",
        "unconfirmedReceived": "0 ETH",
        "unconfirmedSent": "0 ETH",
        "unstaked": "0 ETH",
        "unstaking": "0 ETH",
      },
    },
  },
}
`;

exports[`ethereum send transfer integration test Send some bat token to second portfolio 1`] = `
{
  "1_amount": "22771120000000000 base",
  "2_txId": "0xb04e22fe58c5d68826c860ab89fd022a6023be4cf1afbb629cda2b93bf378b75",
  "3_txRawHex": "02f8b00113841dcd6500850bc208d900828aa9940d8775f648430679a709e98d2b0cb6250d2887ef80b844a9059cbb000000000000000000000000f6c138c36341138ddfc314a11038da8264b7ef090000000000000000000000000000000000000000000000000050e63686bce000c080a02d2731eaaa18c361c27575c32a9a24adf27262ed127ecbbc2c9371dd59e40fb9a01e9e7bfd431028fe5b8add1ee2d73b1432f42a41b7151f9fb7bee0a507d9c3bb",
  "4_tx": {
    "coinAmount": "-0.02277112 BAT",
    "coinName": "bat",
    "confirmations": 0,
    "currencies": {
      "bat": {
        "BAT": 18,
        "base": 0,
      },
      "ethereum": {
        "ETH": 18,
        "Gwei": 9,
        "Kwei": 3,
        "Mwei": 6,
        "finney": 15,
        "szabo": 12,
        "wei": 0,
      },
    },
    "data": {
      "gasLimit": 35497,
      "nonce": 19,
      "replacedTxId": undefined,
      "tipGasPrice": "500000000",
    },
    "date": "SOME DATE",
    "feeAmount": "0.0017925985 ETH",
    "feeCoinName": "ethereum",
    "to": "0xf6c138c36341138ddfc314a11038da8264b7ef09",
    "txId": "0xb04e22fe58c5d68826c860ab89fd022a6023be4cf1afbb629cda2b93bf378b75",
    "version": 1,
  },
  "5_beforeBalances": {
    "exodus_0": {
      "bat": {
        "balance": "4.42277112 BAT",
        "spendable": "4.42277112 BAT",
        "spendableBalance": "4.42277112 BAT",
        "staked": "0 BAT",
        "staking": "0 BAT",
        "total": "4.42277112 BAT",
        "unconfirmedReceived": "0 BAT",
        "unconfirmedSent": "0 BAT",
        "unstaked": "0 BAT",
        "unstaking": "0 BAT",
      },
      "ethereum": {
        "balance": "0.004228230740891449 ETH",
        "spendable": "0.004228230740891449 ETH",
        "spendableBalance": "0.004228230740891449 ETH",
        "staked": "0 ETH",
        "staking": "0 ETH",
        "total": "0.004228230740891449 ETH",
        "unconfirmedReceived": "0 ETH",
        "unconfirmedSent": "0 ETH",
        "unstaked": "0 ETH",
        "unstaking": "0 ETH",
      },
    },
    "exodus_1": {
      "bat": {
        "balance": "0 BAT",
        "spendable": "0 BAT",
        "spendableBalance": "0 BAT",
        "staked": "0 BAT",
        "staking": "0 BAT",
        "total": "0 BAT",
        "unconfirmedReceived": "0 BAT",
        "unconfirmedSent": "0 BAT",
        "unstaked": "0 BAT",
        "unstaking": "0 BAT",
      },
      "ethereum": {
        "balance": "0.005932010192264297 ETH",
        "spendable": "0.005932010192264297 ETH",
        "spendableBalance": "0.005932010192264297 ETH",
        "staked": "0 ETH",
        "staking": "0 ETH",
        "total": "0.005932010192264297 ETH",
        "unconfirmedReceived": "0 ETH",
        "unconfirmedSent": "0 ETH",
        "unstaked": "0 ETH",
        "unstaking": "0 ETH",
      },
    },
  },
  "6_afterBalances": {
    "exodus_0": {
      "bat": {
        "balance": "4.4 BAT",
        "spendable": "4.4 BAT",
        "spendableBalance": "4.4 BAT",
        "staked": "0 BAT",
        "staking": "0 BAT",
        "total": "4.4 BAT",
        "unconfirmedReceived": "0 BAT",
        "unconfirmedSent": "0.02277112 BAT",
        "unstaked": "0 BAT",
        "unstaking": "0 BAT",
      },
      "ethereum": {
        "balance": "0.002435632240891449 ETH",
        "spendable": "0.002435632240891449 ETH",
        "spendableBalance": "0.002435632240891449 ETH",
        "staked": "0 ETH",
        "staking": "0 ETH",
        "total": "0.002435632240891449 ETH",
        "unconfirmedReceived": "0 ETH",
        "unconfirmedSent": "0.0017925985 ETH",
        "unstaked": "0 ETH",
        "unstaking": "0 ETH",
      },
    },
    "exodus_1": {
      "bat": {
        "balance": "0 BAT",
        "spendable": "0 BAT",
        "spendableBalance": "0 BAT",
        "staked": "0 BAT",
        "staking": "0 BAT",
        "total": "0 BAT",
        "unconfirmedReceived": "0 BAT",
        "unconfirmedSent": "0 BAT",
        "unstaked": "0 BAT",
        "unstaking": "0 BAT",
      },
      "ethereum": {
        "balance": "0.005932010192264297 ETH",
        "spendable": "0.005932010192264297 ETH",
        "spendableBalance": "0.005932010192264297 ETH",
        "staked": "0 ETH",
        "staking": "0 ETH",
        "total": "0.005932010192264297 ETH",
        "unconfirmedReceived": "0 ETH",
        "unconfirmedSent": "0 ETH",
        "unstaked": "0 ETH",
        "unstaking": "0 ETH",
      },
    },
  },
}
`;

exports[`ethereum send transfer integration test Send some eth from first to second portolio 1`] = `
{
  "1_amount": "140891449 wei",
  "2_txId": "0x68752c17b5b555623c47bf20a1b41447fad99944540c8d0e8e84efe59564b710",
  "3_txRawHex": "02f86f0113841dcd6500850bc208d90082520894f6c138c36341138ddfc314a11038da8264b7ef09840865d53980c001a0af54c84b5c4a8020ce825633a395aca35428254c8f379285b83ab3109fc261f9a051f41bde79f956b4128ecf12659a37f8092e1757e84eb2cb75746a29c5d813be",
  "4_tx": {
    "coinAmount": "-0.000000000140891449 ETH",
    "coinName": "ethereum",
    "confirmations": 0,
    "currencies": {
      "ethereum": {
        "ETH": 18,
        "Gwei": 9,
        "Kwei": 3,
        "Mwei": 6,
        "finney": 15,
        "szabo": 12,
        "wei": 0,
      },
    },
    "data": {
      "gasLimit": 21000,
      "nonce": 19,
      "replacedTxId": undefined,
      "tipGasPrice": "500000000",
    },
    "date": "SOME DATE",
    "feeAmount": "0.0010605 ETH",
    "feeCoinName": "ethereum",
    "to": "0xf6c138c36341138ddfc314a11038da8264b7ef09",
    "txId": "0x68752c17b5b555623c47bf20a1b41447fad99944540c8d0e8e84efe59564b710",
    "version": 1,
  },
  "5_beforeBalances": {
    "exodus_0": {
      "ethereum": {
        "balance": "0.004228230740891449 ETH",
        "spendable": "0.004228230740891449 ETH",
        "spendableBalance": "0.004228230740891449 ETH",
        "staked": "0 ETH",
        "staking": "0 ETH",
        "total": "0.004228230740891449 ETH",
        "unconfirmedReceived": "0 ETH",
        "unconfirmedSent": "0 ETH",
        "unstaked": "0 ETH",
        "unstaking": "0 ETH",
      },
    },
    "exodus_1": {
      "ethereum": {
        "balance": "0.005932010192264297 ETH",
        "spendable": "0.005932010192264297 ETH",
        "spendableBalance": "0.005932010192264297 ETH",
        "staked": "0 ETH",
        "staking": "0 ETH",
        "total": "0.005932010192264297 ETH",
        "unconfirmedReceived": "0 ETH",
        "unconfirmedSent": "0 ETH",
        "unstaked": "0 ETH",
        "unstaking": "0 ETH",
      },
    },
  },
  "6_afterBalances": {
    "exodus_0": {
      "ethereum": {
        "balance": "0.0031677306 ETH",
        "spendable": "0.0031677306 ETH",
        "spendableBalance": "0.0031677306 ETH",
        "staked": "0 ETH",
        "staking": "0 ETH",
        "total": "0.0031677306 ETH",
        "unconfirmedReceived": "0 ETH",
        "unconfirmedSent": "0.001060500140891449 ETH",
        "unstaked": "0 ETH",
        "unstaking": "0 ETH",
      },
    },
    "exodus_1": {
      "ethereum": {
        "balance": "0.005932010192264297 ETH",
        "spendable": "0.005932010192264297 ETH",
        "spendableBalance": "0.005932010192264297 ETH",
        "staked": "0 ETH",
        "staking": "0 ETH",
        "total": "0.005932010192264297 ETH",
        "unconfirmedReceived": "0 ETH",
        "unconfirmedSent": "0 ETH",
        "unstaked": "0 ETH",
        "unstaking": "0 ETH",
      },
    },
  },
}
`;
